addons/contexts/src/preview/api.ts

```diff
@@ -30,8 +30,7 @@ export const addonContextsAPI = singleton(() => {
 
   // (Vue) hold a reference for updating props in its reactive system
   let reactivePropsMap = {};
-  const updateReactiveSystem = (propsMap: GenericObject) =>
-    Object.assign(reactivePropsMap, propsMap);
```

Comment:
If I remembered correctly, this may break Vue reactive system.  I have to give it a look later of the day.

Reply:
I have added a comment around this :)